### PR TITLE
ENYO-5553: Added sample to qa-sampler with many ExpandableLists

### PR DESCRIPTION
**Pull request by LucieRoy**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

File: packages/sampler/stories/qa-stories/Scroller.js [hunk 1]
```diff
@@ -58,7 +60,7 @@ storiesOf('Scroller', module)
 		'Horizontal scroll',
 		() => (
 			<Scroller
-				direction={select('direction', prop.direction, Scroller, 'horizontal')}
+				direction={select('direction', prop.direction, Scroller, 'vertical')}
```
Comment (webOS101) [inline packages/sampler/stories/qa-stories/Scroller.js]: Is there a reason you changed this default?

File: packages/sampler/stories/qa-stories/Scroller.js [hunk 2]
```diff
@@ -81,4 +83,82 @@ storiesOf('Scroller', module)
...
 			</Scroller>
 		)
-	);
+	)
+	.add(
+		'With Many ExpandableList',
+		() => (
+			<Scroller
+				direction={select('direction', prop.direction, Scroller, 'vertical')}
+				focusableScrollbar={boolean('focusableScrollbar', Scroller, false)}
```
Comment (webOS101) [inline packages/sampler/stories/qa-stories/Scroller.js]: Are these extra knobs needed for this test?  What about the action loggers?  It's OK to have them, but if they won't be used in these tests, maybe it'll be less confusing to not have them?

Comment (LucieRoy) [reply packages/sampler/stories/qa-stories/Scroller.js]: I was thinking about it too....I can remove them if it is okay.

File: packages/sampler/stories/qa-stories/Scroller.js (hunk 1 above)
Comment (LucieRoy) [reply packages/sampler/stories/qa-stories/Scroller.js]: Removed direction knob now

File: packages/sampler/stories/qa-stories/Scroller.js (hunk 2 above)
Comment (LucieRoy) [reply packages/sampler/stories/qa-stories/Scroller.js]: removed action loggers and horizontalScrollbar.  Kept focusableScrollbar.

**Pull request merged**